lib/xopts.ts

```diff
@@ -91,11 +91,15 @@ export class DefaultXOpts<T extends SignerOrProvider> implements GlobalActions {
   }
 }
 
-export async function createXOpts<T extends SignerOrProvider>(
+export async function CreateXOpts<T extends SignerOrProvider>(
```

Comment:
Why are we using pascal case here? I think we are using camel case for functions everywhere else.

Reply:
Fair point, in my mind it was somewhat like a "constructor"/factory method which would use the class's pascal case - mirroring how you'd call `XOpts.load` with XOpts in pascal case - but I think you're right and it should be camel anyway

Reply:
`XOpts` is the class name, which is why it was in pascal case. I would stick with camel for functions, even if it's a factory-like one.